sixtyfps_runtime/rendering_backends/testing/lib.rs

```diff
@@ -31,7 +31,7 @@ impl sixtyfps_corelib::backend::Backend for TestingBackend {
     }
 
     fn run_event_loop(&'static self, _behavior: sixtyfps_corelib::backend::EventLoopQuitBehavior) {
-        unimplemented!("running an event loop with the testing backend");
+        // Nothing to spin, we return right away. Otherwise tests will hang.
```

Comment:
I think it's better to panic because we are not testing things properly.  

For example, we vould have an event loop that just spings and pass mock times for so so many iterations.
And it should be assumed that the test has setup a timer or posted events.

That's why if we panic, we know that this is not yet implemented and not wonder why the eventloop don't run with test and how to fix it.

Reply:
Ok, I've reverted that and in exchange added a `no_run` to one of the Timer doc tests that ended up calling it.